### DISCUSSION TITLE
Welcome to StyleTTS2 Discussions!

### DISCUSSION THREAD

**Discussion by yl4579** [yl4579/StyleTTS2#63] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Inferencer** [comment]
I do HQ lip sync's, maybe it would be worth creating a Discord server for discussions?
https://streamable.com/s6wgqr

**Reply by platform-kit** [reply]
@Inferencer what model are you using for the lipsyncs?

**Reply by Inferencer** [reply]
DINet but I do other stuff to the results in post

**Reply by yl4579** [reply]
I would recommend you join the slack server and we can create a channel for this purpose: https://join.slack.com/t/styletts2/shared_invite/zt-2805io6cg-0ROMhjfW9Gd_ix_FJqjGmQ
